services/discovery/template.yaml

```diff
@@ -18,7 +18,7 @@ Resources:
     Properties:
       CodeUri:
         Bucket: !Sub 'cz-provision-account-${AWS::Region}'
-        Key: latest/services/discovery.zip
+        Key: v1.0.dev/services/discovery.zip
```

Comment:
is this OK?

Reply:
needs to be `latest`; i was just doing some testing ... it's back now